packages/components/package.json

```diff
@@ -39,6 +39,7 @@
         "react-error-boundary": "^4.0.11"
     },
     "devDependencies": {
+        "jest-expo": "~49.0.0",
```

Comment:
Can this be set to something less "chronicles-dependent" like "*"?
It will be a little bit hard to maintain this dependency whenever we bump the expo version of Chronicles

Reply:
Tried changing it to "*", but then the test fails again:

![image](https://github.com/equinor/mad/assets/83707387/a0414f1a-699b-4625-b17a-1b22b8aae9ce)